qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml

```diff
@@ -344,7 +344,7 @@
         </module>
         <module name="LineLength">
             <property name="max" value="80"/>
-            <property name="ignorePattern" value="^import .*$"/>
+            <property name="ignorePattern" value="^import .*$|^\s+\*.*https?:\/\/.+"/>
```

Comment:
@JimDeanSpivey yes, but mine will also, but yours will work for:
```
/**
 *Some text <a href="http://google.com/path?longurlisveeeeeerrrrrrrryyyyylong=1">http://google.com</a>
 * with a link in it.
 */
```
Which is not correct (until we get some javadoc validation at least).